app/views/markets/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_for @market, url: "./" do |f| %>
```

Comment:
If you've set up your routes RESTfully, you can get away without providing a URL for `form_for`s.